WordProject/src/wordCount/Main.java

```diff
@@ -0,0 +1,72 @@
+
+		String newText=oldText.toLowerCase();
+
+		String unText=newText.replaceAll("[[\\.\\?\\!\\,\\;\\:\\{\\}\\(\\)\\']]", "");
+
+		Scanner input=new Scanner(unText);
+
+		ArrayList<String> words = new ArrayList<String>();
+
+		ArrayList<Integer> count= new ArrayList<Integer>();
+
+		while(input.hasNext())
+		{
+			//get next
+			String nextWord = input.next();
+			// if word is in ArrayList
+			if(words.contains(nextWord))
+			{
+				int index = words.indexOf(nextWord);
+				count.set(index, count.get(index)+1);
+			}
+			else
+			{
+				words.add(nextWord);
+				count.add(1);
+			}
+
+		}
+
+		input.close();
+		//print result
+
+		for(int i=0; i<words.size();i++)
+		{
+			// Collections.sort(words);
+			// Collections.reverse(words);
```

Comment:
Line 44 - Don't forget to limit your loop iterations to `< 50` as per the instructions.